public/app/features/alerting/unified/components/rule-editor/EvaluationGroupQuickPick.tsx

```diff
@@ -6,7 +6,8 @@ import { Button, Stack } from '@grafana/ui';
 
 import { formatPrometheusDuration, parsePrometheusDuration, safeParsePrometheusDuration } from '../../utils/time';
 
-export const getEvaluationGroupOptions = (minInterval = '10s') => {
+const MIN_INTERVAl = config.unifiedAlerting.minInterval ?? '10s';
```

Comment:
Do we want nullish coalescing here really? Is it possible for this to come through as `''`, which would then persist rather than falling back?

Reply:
Great question, I'll give this a try and see if it works to manually specify `minInterval: ""` but I suspect the instance will fail to boot because that's not a valid duration.